packages/editor/src/components/editor-help/index.native.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { kebabCase } from 'lodash';
+import { paramCase as kebabCase } from 'change-case';
```

Comment:
Can any of these uses (labels, as far as I can tell) be non-ASCII? I remember this caused issues before.

Reply:
Good call, @sgomes! They are localized strings, so they can definitely contain non-ASCII characters. 

I've pushed https://github.com/WordPress/gutenberg/pull/48776/commits/e002a14ed643e8db2efd8857b66f32b51c792bad to address this.

This is the same approach we used in #45281, so it is battle-tested (on top of the existing unit tests). 

Let me know what you think.